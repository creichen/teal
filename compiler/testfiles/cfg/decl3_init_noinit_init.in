fun f() = {
  var x := 1;
  var y;
  var z := 2;
}

