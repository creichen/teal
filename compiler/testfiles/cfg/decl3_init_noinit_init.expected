Entry#0 @ 1 :
  -> IntConstant#0
VarDecl#0 @ 2 :
  -> VarDecl#1
IntConstant#0 @ 2 :
  -> VarDecl#0
VarDecl#1 @ 3 :
  -> IntConstant#1
VarDecl#2 @ 4 :
  -> Exit#0
IntConstant#1 @ 4 :
  -> VarDecl#2
Exit#0 @ 5 :
